Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -694,6 +694,14 @@ def sigmaAssoc {α : Type*} {β : α → Type*} (γ : ∀ a : α, β a → Type*
   left_inv _ := rfl
   right_inv _ := rfl
 
+/-- Dependent product of sorts is associative up to an equivalence. -/
```

Comment:
Tiny detail: in my book, this is not a complete sentence, so should be rephrased before this ends with a period. How about this? (The same applies to `sigmaAssoc`.)
```suggestion
/-- The dependent product of sorts is associative up to an equivalence. -/